chapters/connectors.tex

```diff
@@ -903,24 +905,58 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
+When generating connection graph equations, only the overdetermined components of a connector are considered.
+The connection graph equations replace the equality-equations for variables that are neither flow nor stream in \cref{generation-of-connection-equations}.
+
+\subsubsection{Remove Redundant Connect-Equations}
+In order to eliminate any redundant \lstinline!connect!-equation the following preparation is needed.
+\begin{nonnormative}
+In the common case where there are no redundant \lstinline!connect!-equations a \lstinline!connect! between connectors with \emph{one} overdetermined component will directly generate \emph{one} optional spanning-tree edge.
+\end{nonnormative}
+\begin{enumerate}
+\item
+  The connection sets are built similarly to the normal way, but keeping the overdetermined components as primitives.
+\item
+  Instead of generating the equality-equation for an overdetermined component, an optional spanning-tree edge in the virtual connection graph is constructed.
+\end{enumerate}
+If a connection set contains $n$ overdetermined components, and was built from $m$ \lstinline!connect!-equations, then $m-n-1$ of the \lstinline!connect!-equations are redundant.
+If there are no redundant \lstinline!connect!-equations (i.e., if $m=n-1$), the optional spanning tree edges can be chosen to correspond to the \lstinline!connect!-equations for overdetermined connectors.
+If there are redundant \lstinline!connect!-equations there will always exist \lstinline!connect!-equations without a corresponding optional spanning tree edge.
+\begin{nonnormative}
+They are called redundant since if they were removed the same connection set and equations would be generated.
```

Comment:
In terms of redundancy:
```suggestion
It is called \emph{redundancy} since this number \lstinline!connect!-equations could be removed without changing the connection set or the generated equations.
```